lib/index.js

```diff
@@ -0,0 +1,137 @@
+"use strict";
```

Comment:
Please take look at  official translations 
https://github.com/marmelab/react-admin/tree/master/packages/ra-language-french
https://github.com/marmelab/react-admin/tree/master/packages/ra-language-english
these repository have only "src/index.ts" file, removed other files

thanks you

Reply:
The published npm package have these files

Can you please elaborate more on the problem here

Reply:
I really don't understand the problem here, my changes are based on the link you're referring and everything works fine with me